src/i18n/ru/index.js

```diff
@@ -0,0 +1,155 @@
+import potato from './potato';
+import print from './print';
+import tester from './tester';
+export default {
+  en: {
```

Comment:
change this label to 'ru' please and then it will be picked up automatically.

Reply:
@yanfali Oops! Overlooked it, sorry! Fixed now.